src/components/TextEditor/extensions/table/TableCellActionHandle.vue

```diff
@@ -0,0 +1,59 @@
+<template>
```

Comment:
the visibility of this button is annoying, it is shown always, it should probably be shown only when a table cell is active